### ISSUE TITLE
The description for the delete command is not specific enough for deleting by uid which causes a mismatch of user expectations.

### ISSUE THREAD

**Issue by bennyLCK** [bennyLCK/ped#8]
![image.png](https://raw.githubusercontent.com/bennyLCK/ped/main/files/e5a4777b-23f1-43cd-ae35-0268e0a9d658.png)

Based on the current description for the delete command, I would expect to run the `delete 101` command to delete a person whose uid is 101, in line with the rest of the delete commands. But when I ran the command, the error message shown was that an invalid index was provided which is not what I would expect if there was some syntax issue regarding supplying the uid :

![image.png](https://raw.githubusercontent.com/bennyLCK/ped/main/files/d525d0cf-f085-4650-b8c8-ac80b92a6b47.png)

However, I could only actually delete someone by uid when I use the syntax `delete uid/101` and this syntax was not specified in the UG.

To sum it up, the only fix needed in this case is to update the current line, `Format: delete INDEX / delete UID / delete NAME` in the UG under delete command to something like this `Format: delete INDEX / delete uid/UID / delete NAME`. But if you do so, please discuss as a team whether this may itself be flagged as a feature flaw later on as well since the command format for `delete` is now not uniform and consistent.
